addons/l10n_si/data/account.account.tag.csv

```diff
@@ -1,15 +1,29 @@
 "id","name","applicability"
-"DDV_I_0","Izstopni DDV 0%","taxes"
-"DDV_I_9_5","Izstopni DDV 9.5%","taxes"
-"DDV_I_22","Izstopni DDV 22%","taxes"
-"DDV_V_0","Vstopni DDV 0%","taxes"
-"DDV_V_9_5","Vstopni DDV 9.5%","taxes"
-"DDV_V_22","Vstopni DDV 22%","taxes"
-"DDV_V_9_5_NE","Vstopni DDV 9.5% - neodbitni","taxes"
-"DDV_V_22_NE","Vstopni DDV 22% - neodbitni","taxes"
-"DDV_V_9_5_SO","Vstopni DDV 9.5% - SO","taxes"
-"DDV_I_9_5_SO","Izstopni DDV 9.5% - SO","taxes"
-"DDV_V_9_5_sam","Vstopni DDV 9.5% - samoobdavčitev","taxes"
-"DDV_V_22_SO","Vstopni DDV 22% - SO","taxes"
-"DDV_I_22_SO","Izstopni DDV 22% - SO","taxes"
-"DDV_V_22_sam","Vstopni DDV 22% - samoobdavčitev","taxes"
+"tag_si_01","Znesek osnov za DDV","taxes"
+"tag_si_02","- neodbitni DDV","taxes"
+"tag_si_03","- vstopni DDV","taxes"
+"tag_si_04","Nabave po osnovni stopnji DDV","taxes"
```

Comment:
You should give better ids and names to your tags, so that one can identify more easily which line of which report it corresponds to. 

For example, let's consider tag tag_si_04. It corresponds to report line financial_report_line_si_01010101. You should change the tag to include some of the line's data, this way: 
- id="vat_tag_01010101"
- name="Slovenia VAT Form: Nabave po osnovni stopnji DDV"